content/en/network_monitoring/performance/network_map.md

```diff
@@ -61,6 +67,20 @@ Click on a node and select _Inspect_ from the menu to contextualize it within th
 
 {{< img src="network_performance_monitoring/network_map/network_entity_zoom.mp4" alt="Network entity zoom" video="true" width="70%">}}
 
+## Map clusters
+
+For complex networks, the map's query editor includes additional grouping fields. This enables you to render datasets that would otherwise have too many nodes to show at once on the map, as well as improve the performance of high cardinality queries.
```

Comment:
```suggestion
For complex networks, the map's query editor includes additional grouping fields. This enables you to render datasets that would otherwise have too many nodes to show at once on the map. Using the additional grouping fields also improves the performance of high cardinality queries.
```